nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/settings/ui/flow-analysis-rules/flow-analysis-rule-table/flow-analysis-rule-table.component.html

```diff
@@ -0,0 +1,145 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="relative h-full border">
+    <div class="flow-analysis-rule-table listing-table absolute inset-0 overflow-y-auto">
+        <table mat-table [dataSource]="dataSource" matSort matSortDisableClear>
+            <!-- More Details Column -->
+            <ng-container matColumnDef="moreDetails">
+                <th mat-header-cell *matHeaderCellDef></th>
+                <td mat-cell *matCellDef="let item">
+                    <ng-container *ngIf="canRead(item)">
+                        <div class="flex items-center gap-x-3">
+                            <div class="pointer fa fa-book" title="Usage"></div>
+                            <!-- TODO - handle read only in configure component? -->
+                            <div *ngIf="hasComments(item)">
+                                <div
+                                    class="pointer fa fa-comment"
+                                    [delayClose]="false"
+                                    nifiTooltip
+                                    [tooltipComponentType]="TextTip"
+                                    [tooltipInputData]="getCommentsTipData(item)"></div>
+                            </div>
+                            <div *ngIf="hasErrors(item)">
+                                <div
+                                    class="pointer fa fa-warning has-errors"
+                                    [delayClose]="false"
+                                    nifiTooltip
+                                    [tooltipComponentType]="ValidationErrorsTip"
+                                    [tooltipInputData]="getValidationErrorsTipData(item)"></div>
+                            </div>
+                            <div *ngIf="hasBulletins(item)">
```

Comment:
I don't believe that FlowAnalysisRules support bulletins.

Reply:
I am not sure. I saw this:

https://github.com/apache/nifi/blob/main/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-flow-analysis-rule.js#L177C2-L177C2

So I kept it...

Reply:
We discussed this and will keep the bulletins for now as it is not clear if this feature supports it or not.